app/lib/actions/katello/host/update.rb

```diff
@@ -30,6 +30,10 @@ def plan(host, consumer_params = nil)
           plan_self(:hostname => host.name)
         end
 
+        def resource_locks
+          :link
```

Comment:
Yeah, i've been asking myself "When should an object actually be locked", and for hosts I have a hard time coming up with many examples aside from keeping actions from occuring while the host is in process of deleting.  For other things I'm not sure what the benefit its?

@iNecas may have some other thoughts about why we use real locks for so many system/host tasks, granted i think previously most things under the /rhsm api did not lock the object.  

I'm wondering on https://github.com/Katello/katello/pull/5656 too if we need to relax the locking as well for content actions on hosts, as I originally saw the issue with UploadPackageProfile because the client tries to upload it after a package install  and the current running package install task had the lock. 

But to go back to your original question, if i'm doing something like installing  a package, should i prevent the user from changing the service level at the same time? (as an example)  

@iNecas is there a way to just lock a task like update from running while another update task (and only an update task) is running?